org.lflang/src/org/lflang/generator/rust/RustValidator.kt

```diff
@@ -103,39 +105,43 @@ class RustValidator(
         @JsonProperty("expansion") val expansion: RustSpanExpansion?
     ) {
         val start: Position
-            get() = Position.fromOneBased(lineStart, columnStart)
+            get() = Position.fromZeroBased(lineStart, columnStart)
```

Comment:
This is a bugfix, previously Cargo messages were not associated with the correct line

Reply:
Thanks for looking into this! I noticed this fixes things in the lfc cli, but maybe this was not the correct fix. I'll investigate tomorrow

Reply:
Interesting -- for the specific program that failed in CI, after reverting this change it looked like the line reported by `lfc` CLI was correct